cosmic-applet-status-area/src/subscriptions/status_notifier_item.rs

```diff
@@ -64,7 +71,9 @@ impl StatusNotifierItem {
 
     // TODO: Only fetch changed part of layout, if that's any faster
     pub fn layout_subscription(&self) -> iced::Subscription<Result<Layout, String>> {
-        let menu_proxy = self.menu_proxy.clone();
+        let Some(menu_proxy) = self.menu_proxy.clone() else {
+            return iced::Subscription::none();
```

Comment:
Not sure what iced subscriptions are, or if it's okay to return none here. I'm gonna have to look into that as well.

Reply:
Thank you. It looks like `Layout` is used for menus only, so `Subscription::none()` can be safely returned here.
I also renamed `layout_subscription` to `menu_layout_subscription` to help clarify this.